engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js

```diff
@@ -31,6 +31,10 @@ angular.module('Bastion.errata').controller('ErrataController',
         $scope.table = nutupane.table;
         $scope.removeRow = nutupane.removeRow;
 
+        Erratum.queryUnpaged({'organization_id': CurrentOrganization}, function (result) {
+            $scope.errataCount = result.total;
+        });
+
```

Comment:
I have many thousands of errata, and this slows the page down about 1000% for me :-(

The RABL generates so many SQL queries.